diagnostic.md

```diff
@@ -94,4 +94,4 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+_'teamMember['hobby'] = "exercise"', 'teamMember['name'] = [object Object] {  given: "Jason", surname: "Weeks"}', and 'teamMember['name']['surname'] = "Weeks"'._
```

Comment:
You're close on the second one.  The value of teamMember['name'] does not include [object Object], but the rest is correct.